.github/workflows/ci.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         java-version: 1.8.0
 
     - run: |
-        export TRAVIS_PULL_REQUEST=${{ github.event.pull_request!! }}
+        export TRAVIS_PULL_REQUEST=${{ (github.event.pull_request!)! }}
```

Comment:
Why are they at the end

Should be !!expr

Reply:
```suggestion
        export TRAVIS_PULL_REQUEST=${{ !!github.event.pull_request }}
```

Reply:
Whoops, fixed that